static/script.js

```diff
@@ -69,3 +70,39 @@ function enableAutoRefresh () { // eslint-disable-line no-unused-vars
     }
   }
 }
+
+function unfoldAll () { // eslint-disable-line no-unused-vars
```

Comment:
unnecessary space between function name and parenthesis 
look at this https://www.w3schools.com/js/js_functions.asp

these are ok `function () {`, `function unfoldAll() { `,
but this is not `function unfoldAll () { `

Reply:
https://github.com/airbnb/javascript#functions--signature-spacing

Reply:
We are using https://standardjs.com/rules.html
Code formatting is just fine.